### PR TITLE
Add the sample of using Crosswalk runtime library with Gradle and Maven

### PR DESCRIPTION
**Pull request by fujunwei**
BUG=XWALK-5744

### CODE REVIEW

File: embedded-maven/README.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+aar is a sample maven project to deal with aar
```
Comment (zqzhang) [inline embedded-maven/README.txt]: Better to use .md format for the README file, which is expected to briefly describe what the sample wants to show and how end user to use it.

**Comment by zqzhang** [comment]
Another, is it OK to use Android logos (images) from license point of view? And there is no license info for the sample code.

**Comment by rakuco** [comment]
+@baleboy for the licensing question

**Comment by fujunwei** [comment]
ping @baleboy @zqzhang

File: embedded-gradle/.gitignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.gradle
```
Comment (zqzhang) [inline embedded-gradle/.gitignore]: Better add these into the `.gitignore` in the root of this project.

File: embedded-gradle/app/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (zqzhang) [inline embedded-gradle/app/.gitignore]: ditto

**Comment by zqzhang** [comment]
Seems you removed the README file from this sample. Then I have no idea how to enable this sample from end user perspective and what it wants to demo.

**Comment by fujunwei** [comment]
There are no README file when creating project in Android Studio.

**Comment by fujunwei** [comment]
Customer need build it in Android Studio or command line to generate a APK for demo.
> Then I have no idea how to enable this sample from end user perspective and what it wants to demo.

File: embedded-gradle/.gitignore (hunk 2 above)
Comment (fujunwei) [inline embedded-gradle/.gitignore]: It's root of project.

**Comment by zqzhang** [comment]
> Customer need build it in Android Studio or command line to generate a APK for demo.

Is it possible to create one README.md like https://github.com/crosswalk-project/crosswalk-samples/blob/master/space-dodge-game/README.md?

**Comment by baleboy** [comment]
Since you are publishing this on Github, a Readme explaining what this source code is for and how it was created is needed (even if the files are meant for a tool and not for a human - people may still stumble in this repository in other ways).

**Comment by baleboy** [comment]
LGTM once the README is added.

**Comment by fujunwei** [comment]
@zqzhang  PTAL.

**Comment by zqzhang** [comment]
LGTM

**Comment by zqzhang** [comment]
@rakuco please help merge this PR, thanks.

**Pull request merged**